### PR TITLE
Geoffreyyoung lgx

### PR DESCRIPTION
**Pull request by geoffrey-young**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?

Info and models for LGX users

#### How was it tested? 

It's on my build

#### Any background context you want to provide?

People still ask where they can find LGX info, especially the cable cover, and I've had these models for a while now so I figured I'd post them.

#### Screenshots (if appropriate)

Images included in PR

### CODE REVIEW

**Comment by geoffrey-young** [comment]
Everything is an original design except for the LGX mount, which is a slight mod of the part posted on discord.  Happy to add more attributions,  remove the mount, or whatever.  Just trying to be a good citizen :))

--Geoff

On Oct 8, 2021, at 3:01 PM, KiloQubit ***@***.***> wrote:

﻿
Did you design these parts, or are these Nemgrea's? If you didn't design them, I'm going to need Nemgrea to comment here that it's ok to check them in with this PR.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by KiloQubit** [comment]
> Everything is an original design except for the LGX mount, which is a slight mod of the part posted on discord. Happy to add more attributions, remove the mount, or whatever. Just trying to be a good citizen :))
> […](#)

Thanks for the clarification, I think the attribution in the README is fine, I will do a full review of your mod later.

File: printer_mods/geoffreyyoung/lgx/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+Bondtech has some official documentation for configuring Klipper [here](https://www.bondtech.se/product-category/upgrade-kits/lgx/).  Below is the configuration I am currently running.
+
+```
+rotation_distance: 8
+microsteps: 16
+full_steps_per_rotation: 200
+
+# use either
+#    gear_ratio: 45:14, 37:17
```
Comment (KiloQubit) [inline printer_mods/geoffreyyoung/lgx/README.md:31]: The first ratio should be 44:14, not 45:14:
Screen Shot 2021-10-08 at 21 25 11

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!  Just a minor fix in the README below.

File: printer_mods/geoffreyyoung/lgx/README.md (hunk 1 above)
Comment (geoffrey-young) [reply printer_mods/geoffreyyoung/lgx/README.md]: done.  thanks!

**Pull request merged**